docs/isolated-cpu-affinity-transition.md

```diff
@@ -0,0 +1,74 @@
+## Isolated CPU affinity transition
+
+The introduction of the kernel commit 46a87b3851f0d6eb05e6d83d5c5a30df0eca8f76
+has affected a deterministic scheduling behavior by distributing tasks
+across CPU cores within a cgroups cpuset. It means that some runc operations
+like `runc exec` might be impacted under some circumstances, by example when
+a container has been created within a cgroup cpuset entirely composed of
+isolated CPU cores usually sets either with `nohz_full` and/or `isolcpus`
+kernel boot parameters.
+
+Some containerized real-time applications are relying on this deterministic
+behavior and uses the first CPU core to run a slow thread while other CPU
+cores are fully used by the real-time threads with SCHED_FIFO policy.
+Such applications can prevent runc process from joining a container when the
+runc process is randomly scheduled on a CPU core owned by a real-time thread.
+
+Runc introduces a way to restore this behavior via a dedicated kernel boot
+parameter:
+
+`runc.exec.isolated-cpu-affinity-transition`
+
+This parameter can take two values:
+
+* `temporary` to temporarily set the runc process CPU affinity to the first
+isolated CPU core of the container cgroup cpuset.
+* `definitive`: to definitively set the runc process CPU affinity to the first
+isolated CPU core of the container cgroup cpuset.
+
+__WARNING:__ `definitive` requires a kernel >= 6.2, also works with RHEL 9 and
+above.
+
+### How it works ?
```

Comment:
nit: extra space

Reply:
I don't get it, could you suggest the patch please ?